packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -412,16 +413,25 @@ export default class OperatorModeStackItem extends Component<Signature> {
     this.containerEl.scrollTop = 0;
   });
 
-  private doCloseAnimation = dropTask(async () => {
-    this.isClosing = true;
-    if (this.itemEl) {
-      await new Promise<void>((resolve) => {
-        scheduleOnce('afterRender', this, this.handleCloseAnimation, resolve);
-      });
-    }
+  private async startAnimation(animationFlag: 'isClosing' | 'isMovingForward') {
+    this[animationFlag] = true;
```

Comment:
is there ever a time that we need to reset this state back to `false`?

Reply:
this seems like an awkward way to set the state. would it not be clearer if we had a single property instead of 2 different properties? when you have 2 separate properties like this it means that you could potentially have them both set to true--what would that even mean?

maybe collapse these 2 properties into a single property like this:
```ts
@tracked private animationType: 'closing' | 'movingForward' | undefined;
```
this way there is a single property that can only be one of these values.

Reply:
Yeah I agree, the card will not perform two animations at a time in that sense - I will refactor that. Thanks @habdelra !